sdk/eventhub/event-hubs/src/index.ts

```diff
@@ -12,7 +12,6 @@ export {
   EventHubClientOptions,
   EventHubConsumerOptions,
   EventHubProducerOptions,
-  RetryOptions,
```

Comment:
We should export `RetryOptions` from `core-amqp` so that the type is visible to our users in order for them to use it

Reply:
yes, this is done - please see https://github.com/Azure/azure-sdk-for-js/pull/4562/files#r309833058

Reply:
I meant that in this file, we should export `RetryOptions` from `core-amqp`

`export { RetryOptions } from "core-amqp";`

https://github.com/Azure/azure-sdk-for-js/pull/4562/files#r309833058 ensures that the consumers of `core-amqp` hasvevisibility to `RetryOptions`
But we still need to re-export it from here so that the consumers of `event-hubs` have visibility to `RetryOptions`